### PR TITLE
Chapter 11: Account activation

### PR DESCRIPTION
**Pull request by longlb-0945**
Screenshot from 2019-09-17 09-10-16

### CODE REVIEW

**Comment by longlb-0945** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (phucmh-1356) [inline app/helpers/account_activations_helper.rb]: xóa file này đi nếu k dùng

File: app/models/user.rb [hunk 2]
```diff
@@ -34,14 +35,35 @@ def remember
...
+  end
+
+  def send_activation_email
+    UserMailer.account_activation(self).deliver_now
+  end
+
+  private
+
+  def downcase_email
+    self.email = email.downcase
```
Comment (phucmh-1356) [inline app/models/user.rb]:
```suggestion
    email.downcase!   
```

File: app/views/user_mailer/password_reset.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<h1>User#password_reset</h1>
+
+<p>
+  <%= @greeting %>, find me in app/views/user_mailer/password_reset.html.erb
```
Comment (phucmh-1356) [inline app/views/user_mailer/password_reset.html.erb]: I18n

**Comment by longlb-0945** [comment]
ready

**Comment by phucmh-1356** [comment]
merged

**Pull request merged**